cmake/verify-toolchain.cmake

```diff
@@ -16,7 +16,11 @@
 # FORMAT=json: Print the output as a json formatted string, useful for Python
 
 # This is the minimum required Zephyr-SDK version which supports CMake package
-set(TOOLCHAIN_ZEPHYR_MINIMUM_REQUIRED_VERSION 0.11.3)
+if("${ARCH}" STREQUAL "arm64")
+  set(TOOLCHAIN_ZEPHYR_MINIMUM_REQUIRED_VERSION 0.12.4)
+else()
+  set(TOOLCHAIN_ZEPHYR_MINIMUM_REQUIRED_VERSION 0.12)
```

Comment:
Can you update the commit message to explain why we're also increasing the SDK version for other arches too?

Reply:
@carlocaione you can just add:
```
Minimal Zephyr SDK v0.12, as agreed in Toolchain WG.
Main reason is added compiler architectures and important inclusion of fixes,
for example qemu and arm fixes for veneers (TFM)
```

Reply:
@MaureenHelm done

Reply:
Thank you